mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -501,17 +501,17 @@ end     # products.each do
 				"google_folder_iam_member":                     ResourceIamMember(IamFolderSchema, NewFolderIamUpdater, FolderIdParseFunc),
 				"google_folder_iam_policy":                     ResourceIamPolicy(IamFolderSchema, NewFolderIamUpdater, FolderIdParseFunc),
 				"google_folder_iam_audit_config":               ResourceIamAuditConfig(IamFolderSchema, NewFolderIamUpdater, FolderIdParseFunc),
-				"google_healthcare_dataset_iam_binding":        ResourceIamBindingWithBatching(IamHealthcareDatasetSchema, NewHealthcareDatasetIamUpdater, DatasetIdParseFunc, IamBatchingEnabled),
```

Comment:
IamBatchingEnabled/IamBatchingDisabled are just named booleans for clarity, defined in https://github.com/hashicorp/terraform-provider-google/blob/main/google/iam_batching.go. I think we can remove those now- `IamWithBatching(true)` or `IamWithBatching` should be clear.

Reply:
That makes sense. IamWithBatching is much clearer.